src/mondeath.cpp

```diff
@@ -648,15 +648,25 @@ void mdeath::broken( monster &z )
     }
     // make "broken_manhack", or "broken_eyebot", ...
     item_id.insert( 0, "broken_" );
-
     item broken_mon( item_id, calendar::turn );
     const int max_hp = std::max( z.get_hp_max(), 1 );
     const float overflow_damage = std::max( -z.get_hp(), 0 );
     const float corpse_damage = 2.5 * overflow_damage / max_hp;
     broken_mon.set_damage( static_cast<int>( std::floor( corpse_damage * itype::damage_scale ) ) );
 
     g->m.add_item_or_charges( z.pos(), broken_mon );
-
+	
+	
+//adds ammo drop	
+    const bool dropsammo = z.type->has_flag( MF_DROPS_AMMO ); 
```

Comment:
This is a little bit more readable
```suggestion
    const bool drops_ammo = z.type->has_flag( MF_DROPS_AMMO ); 
```